org.eclipse.mylyn.releng/multipass/roles/jenkins/files/update-center.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this intended?

Reply:
No not really. This was left from a test that you can use an updatesite without redirect to different mirrors.
Will try to delete the changes in the next days.